rotkehlchen/tests/api/test_icons.py

```diff
@@ -91,3 +93,25 @@ def test_upload_custom_icon_errors(rotkehlchen_api_server, file_upload):
         contained_in_msg=f'does not end in any of {",".join(ALLOWED_ICON_EXTENSIONS)}',
         status_code=HTTPStatus.BAD_REQUEST,
     )
+
+
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_refresh_icon(rotkehlchen_api_server):
+    """Test that checks refreshing the icon of an asset works."""
+    # add icon for an asset
+    icon_manager = rotkehlchen_api_server.rest_api.rotkehlchen.icon_manager
+    root_path = Path(os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))))  # noqa: E501
```

Comment:
lol

don't combine `pathlib` code, with `os` code. Use the `.parent` and the `resolve()` fuctionality. Example:

```python
Path(__file__).resolve().parent.parent.parent
```